metadata/Cadastre/land_ownership

```diff
@@ -0,0 +1,89 @@
+# Title
+
+Utah Land Ownership
+
+## ID
+
+e9eba9ba-3950-4b99-89d2-b1d9282133b6
+
+## Brief Summary
+
+The official State of Utah School and Institutional Trust Lands Administration (SITLA) surface land ownership data.
+
+## Summary
+
+Maintained by a joint effort between SITLA, the BLM, and UGRC, this dataset represents current land ownership in the state of Utah, categorized as federal, state, private, or tribal land. In the cases of federal, state, or private land, this dataset specifies the organization or agency that claims stewardship over that area. However, in the case of private land, this dataset does not identify the specific landowner.
+
+## Description
+
+### What is the dataset?
+
+This dataset depicts land ownership status and areas of responsibility for surface lands in the State of Utah. Land ownership is represented as polygons at a scale of 1:24,000.
+
+Standard colors are recommended when displaying this feature class. A layer file containing the standard color scheme is available by contacting SITLA GIS staff directly at the GIS staff from SITLA at [TLA-GIS@utah.gov] | 801-538-5100, or visiting [https://trustlands.utah.gov].
```

Comment:
Standard colors are recommended when displaying or symbolizing this feature class. A layer file containing the standard color scheme is available by contacting SITLA GIS staff directly at the GIS staff from SITLA at [SITLA-GIS@utah.gov] | 801-538-5100, or visiting [https://trustlands.utah.gov].

Reply:
Do we need to update our website assuming that is where that contact info came from? Also we should keep the contact as generic as possible. I'd recommend removing phone numbers.

Reply:
yeah, I can get the website updated this week.

Reply:
Removed the phone number and fixed the email. @ZachBeck Will you send me the link for that layer file once you receive it?

Reply:
Here's the layer file link:
https://sitla.maps.arcgis.com/home/item.html?id=30a09ecd4d304a1880f6f0b2fe47c59c

Reply:
Fantastic, thank you!